commons/pac-batch-commons/src/main/java/com/tmobile/pacman/commons/utils/NotificationPermissionUtils.java

```diff
@@ -38,37 +37,39 @@ public static void triggerNotificationsForPermissionDenied(List<PermissionVH> pe
             LOGGER.error("Error triggering lambda function url, notification request not sent. Error - {}", e.getMessage());
         }
     }
-    private static NotificationBaseRequest getNotificationBaseRequest(PermissionVH permissionVH,String cloudType) {
+    private static NotificationBaseRequest getNotificationBaseRequest(List<PermissionVH> permissionVHList, String cloudType) {
         NotificationBaseRequest notificationBaseRequest = new NotificationBaseRequest();
         notificationBaseRequest.setEventCategory(Constants.NotificationTypes.PERMISSION);
         notificationBaseRequest.setSubject(SUBJECT);
         notificationBaseRequest.setEventCategoryName(Constants.NotificationTypes.PERMISSION.getValue());
-        notificationBaseRequest.setEventName(String.format(String.format(opsEventName, permissionVH.getErrorVH().getType())));
-        notificationBaseRequest.setEventDescription(String.format(String.format(opsEventName, permissionVH.getErrorVH().getType())));
-
-        PermissionNotificationRequest permissionNotificationRequest = getPermissionNotificationRequest(permissionVH,cloudType);
+        notificationBaseRequest.setEventName(String.format(String.format(opsEventName, cloudType)));
+        notificationBaseRequest.setEventDescription(String.format(String.format(opsEventName, cloudType)));
+        notificationBaseRequest.setEventSourceName(cloudType);
+        JSONObject permissionNotificationRequest = getPermissionNotificationRequest(cloudType, permissionVHList);
         notificationBaseRequest.setPayload(permissionNotificationRequest);
         return notificationBaseRequest;
     }
-
-    private static PermissionNotificationRequest getPermissionNotificationRequest(PermissionVH permissionVH,String cloudType) {
-        PermissionNotificationRequest permissionNotificationRequest = new PermissionNotificationRequest();
-        permissionNotificationRequest.setPermission(permissionVH.getErrorVH().getException());
-        permissionNotificationRequest.setCloudType(cloudType);
-        permissionNotificationRequest.setMessage("Unable to collect data due to missing permission");
-        permissionNotificationRequest.setAccountNumber(permissionVH.getAccountNumber());
-        permissionNotificationRequest.setAssetType(permissionVH.getErrorVH().getType());
-        return permissionNotificationRequest;
+    private static JSONObject getPermissionNotificationRequest(String cloudType, List<PermissionVH> permissionVHList) {
```

Comment:
Now I don't see any empty lines... Methods must be separated by single empty line.

Reply:
Check other methods and fix it if possible. Would be nice step toward good looking code :)